x86_64-buildroot-linux-uclibc-gcc: ERROR: unsafe header/library path used in cross-compilation: '-I/usr/local/include'
x86_64-buildroot-linux-uclibc-gcc: ERROR: unsafe header/library path used in cross-compilation: '-I/usr/local/include'

Fix library and include paths
Prevent inclusion of unused libraries in build. 

Upstream: n/a

Signed-off-by: Andreas Ziegler <br025@umbiko.net>
---
--- brutefir.orig/Makefile
+++ brutefir-1.0o/Makefile
@@ -9,8 +9,8 @@
 
 ###################################
 # Where to find libraries, and their header files.
-LIBPATHS	= -L/usr/local/lib
-INCLUDE		= -I/usr/local/include
+LIBPATHS	= -L$(DESTDIR)/lib
+INCLUDE		= -I$(DESTDIR)/include
 ifdef FFTW_PATH
 LIBPATHS	+= -L$(FFTW_PATH)/lib
 INCLUDE		+= -I$(FFTW_PATH)/include
@@ -90,8 +90,9 @@
 ifeq ($(UNAME),Linux)
 LIB_TARGETS	+= alsa.bfio
 endif
-LIB_TARGETS	+= oss.bfio
-LIB_TARGETS	+= jack.bfio
+# OSS and Jack are not used
+# LIB_TARGETS	+= oss.bfio
+# LIB_TARGETS	+= jack.bfio
 endif
 
 # FreeBSD
